packages/insomnia-app/app/network/o-auth-2/grant-authorization-code.js

```diff
@@ -182,3 +207,11 @@ async function _getToken(
 
   return results;
 }
+
+function _base64UrlEncode(string: string): string {
+  return string
```

Comment:
We absolutely should not be using reserved keywords as variable names, very strange that ES Lint didn't complain 🤔 Even if it works, it can be a little confusing.

```suggestion
function _base64UrlEncode(str: string): string {
  return str
```